src/backends/x11/meta-monitor-manager-xrandr.c

```diff
@@ -282,28 +282,45 @@ output_get_underscanning_borders_xrandr (MetaMonitorManagerXrandr *manager_xrand
 }
 
 static gboolean
-output_get_underscanning_xrandr (MetaMonitorManagerXrandr *manager_xrandr,
-                                 MetaOutput               *output)
+output_get_atom_property (MetaMonitorManagerXrandr *manager_xrandr,
+                          MetaOutput *output, const char *propname,
+                          Atom *value)
 {
-  MetaOutputXrandr *output_xrandr = output->driver_private;
+  gboolean exists = FALSE;
   Atom atom, actual_type;
   int actual_format;
   unsigned long nitems, bytes_after;
-  g_autofree unsigned char *buffer = NULL;
-  g_autofree char *str = NULL;
+  unsigned char *buffer;
```

Comment:
This is not my code, actually, so I don't really have an answer  (I didn't even notice it). I assume, though, it's because otherwise `g_free()` would be called, instead of `XFree()` (which seems to be the right thing to do here, since `buffer` will be allocated by Xlib[1])?
[1] ftp://www.x.org/pub/current/doc/man/man3/XFree.3.xhtml